common/src/main/java/com/wynntils/features/user/overlays/ObjectivesOverlayFeature.java

```diff
@@ -186,18 +193,26 @@ public void render(PoseStack poseStack, float partialTicks, Window window) {
 
                 float renderY = offsetY + this.getRenderY();
 
+                final String text = objective.getGoal() + ": " + objective.getScore() + "/" + objective.getMaxScore();
                 FontRenderer.getInstance()
                         .renderAlignedTextInBox(
                                 poseStack,
-                                objective.getGoal() + ": " + objective.getScore() + "/" + objective.getMaxScore(),
+                                text,
                                 this.getRenderX(),
                                 this.getRenderX() + this.getWidth(),
                                 renderY,
-                                0,
+                                this.getWidth(),
                                 this.textColor,
                                 FontRenderer.TextAlignment.fromHorizontalAlignment(this.getRenderHorizontalAlignment()),
                                 this.textShadow);
 
+                float height = FontRenderer.getInstance().calculateRenderHeight(List.of(text), this.getWidth());
```

Comment:
Is this to handle if multi-line support causes objectives to be so long they get multi-line in our overlay too?

Reply:
Yes